src/datatypes/src/types/json_type.rs

```diff
@@ -31,9 +31,12 @@ use crate::scalars::ScalarVectorBuilder;
 use crate::type_id::LogicalTypeId;
 use crate::types::{ListType, StructField, StructType};
 use crate::value::Value;
+use crate::vectors::json::builder::JsonVectorBuilder;
 use crate::vectors::{BinaryVectorBuilder, MutableVector};
 
 pub const JSON_TYPE_NAME: &str = "Json";
+const JSON_PLAIN_FIELD_NAME: &str = "__plain__";
```

Comment:
Do we need to rename it to a name that has lower conflict possibility? e.g. `__json_plain__`
```suggestion
const JSON_PLAIN_FIELD_NAME: &str = "__json_plain__";
```

Reply:
That's good naming. I'll do it in the next PR.